dds_web/api/__init__.py

```diff
@@ -28,8 +28,8 @@
 ####################################################################################################
 
 # Login/access ###################################################################### Login/access #
-api.add_resource(user.Token, "/user/token", endpoint="token")
 api.add_resource(user.EncryptedToken, "/user/encrypted_token", endpoint="encrypted_token")
+api.add_resource(user.RequestMail2fa, "/user/request_mail2fa", endpoint="request_mail2fa")
```

Comment:
I would have it like 
`api.add_resource(user.SecondFactor, "/user/second_factor", endpoint="second_factor")`
and this endpoint would be used by a user to enter mfa value, not request mfa email.